### PR TITLE
multi calls for democracy.votes and staking.controllers

### PR DESCRIPTION
**Pull request by Stefie**

### CODE REVIEW

**Comment by jacogr** [comment]
Nice! Thank you!

**Comment by Stefie** [comment]
Yeah, not a big win for democracy.votes because we still need to map the accountId's. I changed that more because consistency

File: packages/api-derive/src/staking/controllers.ts [hunk 1]
```diff
@@ -11,10 +11,7 @@ import { drr } from '../util/drr';
 
 function allBonds (api: ApiInterface$Rx, stashIds: Array<AccountId>) {
   return combineLatest(
```
Comment (amaurymartiny) [inline packages/api-derive/src/staking/controllers.ts]: can remove combineLatest

Comment (Stefie) [reply packages/api-derive/src/staking/controllers.ts]: Ok, I will add it to my next PR.

**Pull request merged**